ansible/playbooks/reset_morio.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Reset Morio configuration on the EC2 instances
+  hosts: all
+  become: yes
+  become_method: sudo
+  tasks:
+    - name: Make backup directory
+      shell: mkdir -p ~/.moriod
```

Comment:
As a rule of thumb, we should only use `shell` in Ansible when nothing more suitable is available.

In this case, we should use the built-in file module: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html